### ISSUE TITLE
File Operation Error Dragging and Dropping Folders

### ISSUE THREAD

**Issue by PeeDeeWhite** [TestMace/TestMace-issues#45]
Hi there

This is really just for your information as I have not been able to replicate this problem.

I have a Cloud Project with some requests in folders. I created a new folder and tried to drag and drop an existing folder into the new folder and got the following error:

```
File operation error
Some of the project files are locked by another process or you don't have permissions to change these files. Please terminate this process or change permissions and click 'Retry' or click 'Close and Reload Project'.

Files:

---\AppData\Roaming\testmace-config\cloud-project\Project\Application Service
---\AppData\Roaming\testmace-config\cloud-project\Project\Application Service\Valid Requests
```

I closed the application and re-opened it and tried again and it worked.

**Comment by dima11221122** [comment]
Okay, thank you. Have you faced with the problem on windows?

**Comment by PeeDeeWhite** [comment]
> Okay, thank you. Have you faced with the problem on windows?

Yes, the problem in on Windows 10 with version 1.0.0 of TestMace.  I just had the problem again dragging some Request nodes into a new folder.  The Requests are displayed in the correct folder in the UI. Choosing the option to Close and Retry worked.

I have attached a screenshot.
![Test Mace File IO Error](https://user-images.githubusercontent.com/23114514/78768728-3c262c00-7984-11ea-9dc7-63f6e7893e88.png)

**Comment by dima11221122** [comment]
Okay, thank you for the report. We'll try to solve it ASAP
